### PR TITLE
Update UI Design

### PR DESCRIPTION
**Pull request by nitinshr135**
Fixes SPH-1708

### CODE REVIEW

File: packages/frontend/src/components/Modal/extend-modal.tsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+// TODO - TO BE RELEASED NEXT
+
+function ExtendModal({ option }: IProps) {
+  const ModalCntx = useContext<any>(ModalContext)
+  const { setModalOpen, setModalType, setModalOption } = ModalCntx
+  const [year, setYear] = useState<number>(1)
+
+  const totalPrice = Number(option.gasFee) + Number(option.price)
+  // const isLessBalance = userBalance ? totalPrice > Number(userBalance) : false
```
Comment (abhinav-spheron) [inline packages/frontend/src/components/Modal/extend-modal.tsx:23]: Remove if not needed

File: packages/frontend/src/components/UI/search-domain.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+}
+
+function SearchDomain({ showBtn, classname }: IProps) {
+  const navigate = useNavigate()
+  const [searchQuery, setSearchQuery] = useState<string>('')
+  const [inputFocus, setInputFocus] = useState<boolean>(false)
+
+  const inputRef = useRef<HTMLInputElement>(null)
+
+  const handleSearch = () => {
```
Comment (abhinav-spheron) [inline packages/frontend/src/components/UI/search-domain.tsx:20]: Can implement debounce in search

Comment (nitinshr135) [reply packages/frontend/src/components/UI/search-domain.tsx:20]: hitting enter or clicking on `Search` will forward to another page, thus the user has to put their whole filecoin domain name and search. Using debouncing doesn't seem necessary here

Comment (abhinav-spheron) [reply packages/frontend/src/components/UI/search-domain.tsx:20]: Okay, got it. Not required in that case.

**Review by abhinav-spheron — APPROVED** [review]
LGTM

File: packages/frontend/src/pages/DomainDetail/domain-detail.tsx [hunk 3]
```diff
@@ -21,15 +26,21 @@ const DomainDetail = () => {
...
     useState<boolean>(false)
   const [settingContentHash, setSettingContentHash] = useState<boolean>(false)
   const [isSuccesful, setIsSuccesful] = useState<boolean>(false)
   const [isEditMode, setIsEditMode] = useState<boolean>(false)
   const [isControllerEditMode, setIsControllerEditMode] =
     useState<boolean>(false)
-  const [showCopyPopup, setShowCopyPopup] = useState<boolean>(false)
+  const [showCopyPopup, setShowCopyPopup] = useState<{
+    show: boolean
+    type: string
```
[the remark this answers is not in the document]
Comment (nitinshr135) [reply packages/frontend/src/pages/DomainDetail/domain-detail.tsx]: got it :D

**Pull request merged**